ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -1,41 +1,56 @@
+{% if router_pim_sparse_mode is arista.avd.defined %}
 
 ## Router PIM Sparse Mode
 
 ### IP Sparse Mode Information
-{%     if router_pim_sparse_mode.ipv4 is defined and router_pim_sparse_mode.ipv4 is not none %}
-{%         if router_pim_sparse_mode.ipv4.rp_addresses is defined and router_pim_sparse_mode.ipv4.rp_addresses is not none %}
+{%     if router_pim_sparse_mode.ipv4 is arista.avd.defined %}
+{%         if router_pim_sparse_mode.ipv4.rp_addresses is arista.avd.defined %}
 
-### IP Rendez-vous Information
+#### IP Rendez-vous Information
 
-| Rendez-vous Point Address | Group Address |
+| Rendezvous Point Address | Group Address |
 | ------------------------- | ------------- |
 {%             for rp_address in router_pim_sparse_mode.ipv4.rp_addresses | arista.avd.natural_sort %}
 {%                 for group in router_pim_sparse_mode.ipv4.rp_addresses[rp_address].groups | arista.avd.natural_sort %}
 | {{ rp_address }} | {{ group }} |
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
-{%         endif %}
-{%         if router_pim_sparse_mode.ipv4.anycast_rps is defined and router_pim_sparse_mode.ipv4.anycast_rps is not none %}
+{%         if router_pim_sparse_mode.ipv4.anycast_rps is arista.avd.defined %}
 
-### IP Anycast Information
+#### IP Anycast Information
 
-| IP Anycast Address | Other Rendez-vous Point Address | Register Count |
+| IP Anycast Address | Other Rendezvous Point Address | Register Count |
 | ------------------ | ------------------------------- | -------------- |
 {%             for anycast_rp in router_pim_sparse_mode.ipv4.anycast_rps | arista.avd.natural_sort %}
 {%                 for other_anycast_rp_address in router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses | arista.avd.natural_sort %}
-| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {% if router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is defined and router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count is not none %} {{ router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count }} | {% else %} - {% endif %}|
+{%                     set register_count = router_pim_sparse_mode.ipv4.anycast_rps[anycast_rp].other_anycast_rp_addresses[other_anycast_rp_address].register_count | arista.avd.default('-') %}
+| {{ anycast_rp }} | {{ other_anycast_rp_address }} | {{ register_count }} |
 {%                 endfor %}
 {%             endfor %}
 {%         endif %}
+{%     endif %}
+{%     if router_pim_sparse_mode.vrfs is arista.avd.defined %}
+
+#### IP Sparse Mode VRFs
+
+| VRF Name | Rendezvous Point Address | Group Address |
+| -------- | ------------------------- | ------------- |
```

Comment:
Please delete one ```-``` in the second set to allign with the above heading ```Rendezvous Point Address```

Reply:
Corrected